### ISSUE TITLE
Chicken recipes and drops are out of whack with addition of thermal

### ISSUE THREAD

**Issue by meglazero** [NFINIT-Labs/Fantasy-Skies#23]
There are several issues with the resource chickens mod especially because of the quests that are required for being done to complete the quest page. A handful of the chicken spawn eggs are still set to use mekanism/janoeo foundation ingots that with almostunified are unable to be crafted or seemingly obtained at all with the addition of thermal. Nickel, tin, and lead are the main 3 I know of, as they are also required for quests on the page.

Additionally a handful of the config files for the chickens have some weird settings which result in odd drops. The drop type for silver chickens is set to "items:forge:ingots/silver" which is giving all the varieties of silver ingot that are in the modpack, most recipes seem to be forced to or using the thermal variant of silver ingot so changing the drop to "thermal:silver_ingot" might be beneficial. I believe it was tin chickens that was also set to the "items:forge:ingots/tin" setting, and lead and nickel are both set to mekanism I believe, but most other recipes are trying to force the use of thermal:metal_ingot, to the point you can't craft the other ingots even with nuggets.

I tried to use almostunified to force the chicken recipes to update to using thermal, but I cannot figure out a way to force the thermal ingots onto the recipes.

**Comment by meglazero** [comment]
well I found the solution, just have to go into fantasy skies\scripts\fantasy_skies_recipes.zs and edit the actual recipes for the chicken eggs from item:mekanism:ingot_lead and ingot_tin and item:janoeo_foundation:ingot_nickel to tag:items:forge:ingots/nickel, lead, tin respectively. Might be wise if you're going into the recipes to adjust just those to also adjust any of the others that got added from their respective mod to the forge material unless otherwise intended to be a mod gate.

End result looks like this:
```
    //nickel chicken
craftingTable.addShaped("nickel_chicken_spawn_egg", <item:resourcechickens:nickel_chicken_spawn_egg>, [
    [<tag:items:forge:ingots/nickel>, <tag:items:forge:ingots/nickel>, <tag:items:forge:ingots/nickel>], 
    [<tag:items:forge:ingots/nickel>, <item:minecraft:chicken_spawn_egg>, <tag:items:forge:ingots/nickel>],
    [<tag:items:forge:ingots/nickel>, <tag:items:forge:ingots/nickel>, <tag:items:forge:ingots/nickel>]
    ]);

    //tin chicken
craftingTable.addShaped("tin_chicken_spawn_egg", <item:resourcechickens:tin_chicken_spawn_egg>, [
    [<tag:items:forge:ingots/tin>, <tag:items:forge:ingots/tin>, <tag:items:forge:ingots/tin>], 
    [<tag:items:forge:ingots/tin>, <item:minecraft:chicken_spawn_egg>, <tag:items:forge:ingots/tin>],
    [<tag:items:forge:ingots/tin>, <tag:items:forge:ingots/tin>, <tag:items:forge:ingots/tin>]
    ]);

    //Lead chicken
craftingTable.addShaped("lead_chicken_spawn_egg", <item:resourcechickens:lead_chicken_spawn_egg>, [
    [<tag:items:forge:ingots/lead>, <tag:items:forge:ingots/lead>, <tag:items:forge:ingots/lead>], 
    [<tag:items:forge:ingots/lead>, <item:minecraft:chicken_spawn_egg>, <tag:items:forge:ingots/lead>],
    [<tag:items:forge:ingots/lead>, <tag:items:forge:ingots/lead>, <tag:items:forge:ingots/lead>]
    ]);```

**Comment by knoxhack** [comment]
implemented in 1.3.2
